fedimint-escrow-client/src/cli.rs

```diff
@@ -57,18 +57,11 @@ pub(crate) async fn handle_cli_command(
             // finalize_and_submit txns to lock ecash by underfunding
             // how to call this method?
             let (operation_id, out_point, escrow_id) = escrow
-                .buyer_txn(
-                    Amount::from_sat(cost),
-                    buyer,
-                    seller,
-                    arbiter,
-                    retreat_duration,
-                )
+                .buyer_txn(Amount::from_sat(cost), seller, arbiter, retreat_duration)
                 .await?;
             // even though unique transaction id will be assigned, escrow id will used to
             // collectively get all data related to the escrow
-            pub const CODE: String =
-                hash256((vec![buyer, seller, arbiter, cost].concat()).reverse());
+            pub const CODE: String = hash256((vec![seller, arbiter, cost].concat()).reverse());
```

Comment:
calculating code here seems out of place. Shouldn't the previous operation do it?

The higher level code (this fn) should ideally just call some functions in a simple sequence.

Reply:
Why is this a `const` anyway? And why `reverse()`? Also  - name should explain or at least hint what is this secret for.